jmx-metrics/README.md

```diff
@@ -127,6 +127,25 @@ mutually exclusive with `otel.jmx.groovy.script`. The currently supported target
 - `otel.instrument(MBeanHelper mBeanHelper, String name, String description, String attribute, Closure instrument)` - `unit` is "1" and `labelFuncs` are empty map.
 - `otel.instrument(MBeanHelper mBeanHelper, String name, String attribute, Closure instrument)` - `description` is empty string, `unit` is "1" and `labelFuncs` are empty map.
 
+In cases where you'd like to share instrument names while creating datapoints for multiple MBean attributes:
+
+- `otel.instrument(MBeanHelper mBeanHelper, String instrumentName, String description, String unit, Map<String, Closure> labelFuncs, Map<String, Map<String, Closure>> attributeLabelFuncs, Closure instrument)`
+
+- An example of this would be with tomcat, To be able to consolidate different thread types into one metric:
+
+  ```groovy
+    otel.instrument(beantomcatconnectors, "tomcat.threads", "description", "1",
```

Comment:
some suggested rewording after searching github for the connectors object:

```suggestion
- An example of this in Tomcat, is to consolidate different thread types into one `"tomcat.threads"` metric using both `currentThreadCount` and `currentThreadsBusy` MBean attributes, labeling with their applicable `"Thread Type"`:

  ```groovy
    otel.instrument(otel.mbean("Catalina:type=ThreadPool,name=*"), "tomcat.threads", "description", "1",
```